wolfssl/wolfcrypt/wc_port.h

```diff
@@ -160,9 +160,14 @@
             #define WIN32_LEAN_AND_MEAN
         #endif
         #if !defined(WOLFSSL_SGX) && !defined(WOLFSSL_NOT_WINDOWS_API)
-            #define _WINSOCKAPI_ /* block inclusion of winsock.h header file */
+            #define _WINSOCKAPI_ /* block inclusion of winsock.h header file. */
             #include <windows.h>
+            /* winsock2.h expects _WINSOCKAPI_ to be * undefined, and defines
```

Comment:
The comment looks wrong - star in the middle of line.

Reply:
good catch.  fixed.